gatsby/content/projects/2017-05-02-thunderbird.mdx

```diff
@@ -3,14 +3,34 @@ layout: projectimage
 title: Mozilla Thunderbird
 categories: 
  - client
-description: A free email & chat app
-author: Frédéric Wang
+description: A free email, calendar & chat app
+author: Mozilla
 maturity: Beta
 language: C
 license: MPL-2.0
-repo: 
+platforms:
+    - Linux
+    - Mac
+    - Windows
+repo: https://hg.mozilla.org/comm-central
+home: https://www.thunderbird.net/
+room: "#maildev:mozilla.org"
+features:
```

Comment:
Where did you copy this feature list from? It is not as populated as the one for e.g. Fractal and Thunderbird doesn’t show up in my local build in the [clients matrix](https://matrix.org/clients-matrix) so I’m guessing there isn’t enough information here.

Reply:
> Where did you copy this feature list from? It is not as populated as the one for e.g. Fractal and Thunderbird doesn’t show up in my local build in the [clients matrix](https://matrix.org/clients-matrix) so I’m guessing there isn’t enough information here.

I copied it from `element` and then removed the features that aren't supported. (It is super unclear what the options are?)

Reply:
> Thunderbird doesn’t show up in my local build in the [clients matrix](https://matrix.org/clients-matrix) so I’m guessing there isn’t enough information here.

I think the only things showing up there are the ones moved to `gatsby/content/projects/clients`? I'm not sure though.

I was pointed to https://matrix.org/docs/projects/client/mozilla-thunderbird directly and noted it was very out-of-date.

Reply:
Ha, could be. @benparsons can you confirm? Maybe this update would be a good opportunity to move this to the subdirectory.

Reply:
I can definitely add more, I'll try to see if there's a full list somewhere and not just leave them out?

Reply:
[`src/pages/clients-matrix.js`](https://github.com/matrix-org/matrix.org/blob/33914ac58515b2e26af8402aab8c3aef13a3eaeb/gatsby/src/pages/clients-matrix.js#L225) has the full list.

I reckon having the full list in a file with values as `no` makes it easier to update them at a later point when one reopens the file and notices the feature is now available.

Reply:
Thanks. I've added those. I'll see about moving it / getting some screenshots.

Reply:
`/clients` contains clients with `featured: true` in their yaml. Please move the file to the clients subdirectory as they are updated (can remove the date from the filename too)